test/cache-driver.js

```diff
@@ -35,7 +35,7 @@ var testRequire = util.testRequire;
 var assert = require('assert');
 
 var MemoryCache = testRequire('memory-cache');
-var CacheDriver = testRequire('cache-driver');
+var CacheDriver = testRequire('../lib/cache-driver');
```

Comment:
Why is this necessary.  The whole point of the testRequire function was to avoid having to do this. :)